spec/lib/catalog/name_adjust_spec.rb

```diff
@@ -6,16 +6,39 @@
     context "when passing in multiple Copy Names" do
       let(:names) { [item_name.to_s, "Copy of #{item_name}", "Copy (1) of #{item_name}"] }
 
-      it "returns the highest index in the array" do
-        expect(create_copy_name).to eq "Copy (2) of #{item_name}"
+      context "when the copied name would be greater than 64 characters" do
+        let(:item_name) { "CopiedNamesThatAreGreaterThanSixtyFourCharactersLongWillBeTruncated"}
+
+        it "returns the highest index in the array truncated to 64 characters" do
+          expect(create_copy_name).to eq "Copy (2) of #{item_name}".truncate(64)
```

Comment:
I *think* if we had rubocop on this repo it would complain about not using parenthesis for argument parameters. It's probably one of the main "un-idiomatic ruby" things we do.

Although, it was `eq "Copy (2) of #{item_name}"` before, so I'm fine with leaving it like this since it's at least consistent within the file.